ntbs-service/Pages/Index.cshtml

```diff
@@ -2,13 +2,9 @@
 @model IndexModel
```

Comment:
Not 100% necessary to handle within this commit, but are you aware whether SearchOld/Index.cshtml is still relevant?

Reply:
It's being kept in the codebase for the future so I've moved it to SearchOld for now - something to keep in mind if we don't end up reusing it though